README.md

```diff
@@ -408,6 +409,10 @@ Set up your GitHub Actions workflow with a specific version of your programming
 
 - [GitHub Actions Branding Cheat Sheet](https://haya14busa.github.io/github-action-brandings/)
 
+### Tools
+
+- [Run GitHub Actions Locally](https://github.com/nektos/act)
```

Comment:
It looks like this action may already be in the GitHub Tools and Management Section under "Run GitHub Actions Locally with a web interface". Would you mind providing additional info on how this action is different and moving it to that section?

Reply:
@jessicalostinspace - i moved it under that same section.  fwiw, I originally added this tool over a year ago in #53